src/libraries/TwabLib.sol

```diff
@@ -493,18 +567,14 @@ library TwabLib {
       return prevOrAtObservation;
     }
 
-    // If there is only 1 actual observation, either return that observation or a zeroed observation
+    // If there is only 1 observation and it's after the target, then return zero
     if (_accountDetails.cardinality == 1) {
-      if (_targetTime >= prevOrAtObservation.timestamp) {
```

Comment:
We removed this condition cause it's supposed to be handled in the condition above right?